src/actions/posts.test.js

```diff
@@ -293,9 +294,9 @@ describe('Actions.Posts', () => {
         assert.ok(!reactions[post1.id]);
     });
 
-    it('removePost', async () => {
+    it.only('removePost', async () => {
```

Comment:
Could you remove the `.only` here to ensure all the tests in this file are ran?
```suggestion
    it('removePost', async () => {
```

Reply:
oops! My bad:( will do that!